docs/getting_started/conda.md

```diff
@@ -0,0 +1,127 @@
+# Running HippUnfold with Conda
+
+HippUnfold can be installed and run using Conda on **Linux and macOS** systems.
+
+**Note:** Conda installation is **not supported on Windows** at this time. If you are on Windows, please refer to the [Docker instructions](docker.md) instead.
+
+---
+
+## For Users: Installing HippUnfold via Conda
+
+These steps are intended for **end users** who simply want to run HippUnfold for hippocampal segmentation and unfolding.
+
+### 1. Install Miniconda (if not already installed)
+
+Follow the instructions at the official Miniconda site:  
+[https://www.anaconda.com/docs/getting-started/miniconda/install](https://www.anaconda.com/docs/getting-started/miniconda/install)
+
+---
+
+### 2. Create and activate a new Conda environment
+
+```bash
+conda create --name hippunfold-env -c conda-forge -c bioconda hippunfold
+conda activate hippunfold-env
+```
+
+---
+
+### 3. Test the installation
+
+Run the following command to verify the installation:
+
+```bash
+hippunfold -h
+```
+
+You should see a help message listing all available command-line options.  
+If this runs successfully, you’re ready to start processing data with HippUnfold!
+
+## Running an example
+
+You can try HippUnfold on a sample dataset to make sure everything works as expected.
+
+First, download and extract a single-subject BIDS dataset for this test:
+
+```bash
+wget https://www.dropbox.com/s/mdbmpmmq6fi8sk0/hippunfold_test_data.tar 
+tar -xvf hippunfold_test_data.tar
+```
+
+This will create a `ds002168/` folder with a single subject that has both T1w and T2w images:
+
+```
+ds002168/
+├── dataset_description.json
+├── README.md
+└── sub-1425
+    └── anat
+        ├── sub-1425_T1w.json
+        ├── sub-1425_T1w.nii.gz
+        ├── sub-1425_T2w.json
+        └── sub-1425_T2w.nii.gz
+
+2 directories, 6 files
+```
+
+Now let’s run HippUnfold using the T1w modality:
+
+```bash
+hippunfold ds002168 ds002168_hippunfold participant --modality T1w --cores all --use-conda
+```
+
+This should run the full pipeline and place results in a new `ds002168_hippunfold/` folder.
+
+---
+
```

Comment:
Wondering if we should mention that hippunfold will download a number of things into the app cache dir (in home dir by default), which can be overriden by using the env var